book.rb

```diff
@@ -10,4 +10,13 @@ def initialize(title, author)
   def add_rental(person, date)
     @rentals << Rental.new(person, date, self)
   end
+
+  def to_json(*_args)
+    {
+      'title' => @title,
+      'author' => @author,
+      'rentals' => @rentals,
+      'class' => self.class.name
+    }
+  end
 end
```

Comment:
- [ ] According to the project requirement 👇🏻 Kindly ensure you properly handle the scenario where the storage files are created and data stored if they do not exist. Actually, I'm getting an error when I delete one of the JSON files (the app should work and show no errors).


![11](https://user-images.githubusercontent.com/79058364/217628858-152ad888-0097-4127-919e-9859066ba85c.png)